extensions/theme-monokai/themes/monokai-color-theme.json

```diff
@@ -99,7 +99,8 @@
 		"terminal.ansiBrightBlue": "#819aff", // hue shifted #AE81FF
 		"terminal.ansiBrightMagenta": "#AE81FF",
 		"terminal.ansiBrightCyan": "#66D9EF",
-		"terminal.ansiBrightWhite": "#f8f8f2"
+		"terminal.ansiBrightWhite": "#f8f8f2",
+		"terminal.inactiveSelectionBackground": "#878b9140"
```

Comment:
Was this one needed? I think it should work without this since it doesn't define `editor.inactive...`?

Reply:
Thanks for the catch. Removed.